rmw_connext_shared_cpp/src/qos.cpp

```diff
@@ -26,14 +26,18 @@ namespace
 {
 
 bool
-is_time_default(const rmw_time_t & time)
+is_time_unspecified(const rmw_time_t & time)
 {
+  return rmw_time_equal(time, RMW_DURATION_UNSPECIFIED);
   return time.sec == 0 && time.nsec == 0;
```

Comment:
This is a dead line of code now, is it not?

Reply:
good catch!